sources/camunda-docker-compose-core.yml

```diff
@@ -91,9 +102,24 @@ services:
     networks:
       - camunda-platform
 
+  kibana:
+    image: docker.elastic.co/kibana/kibana:${ELASTIC_VERSION:-7.17.0}
```

Comment:
`ELASTIC_VERSION` here and 78 are different

Reply:
This docker-compose comes from the below source. We are not changing it just consuming it unless there is something really needed to be changed. 
https://github.com/camunda/camunda-platform/tree/8.1.2